Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -2048,6 +2056,32 @@ private void ProcessInteractions(ref TriggerState trigger, int interactionStartI
             }
         }
 
+        private void ProcessHoldInteractions(ref TriggerState trigger, int interactionStartIndex, int interactionCount)
+        {
+            var context = new InputInteractionContext
+            {
+                m_State = this,
+                m_TriggerState = trigger
+            };
+
+            for (var i = 0; i < interactionCount; ++i)
+            {
+                var index = interactionStartIndex + i;
+                var state = interactionStates[index];
+                var interaction = interactions[index];
+                if (interaction is not HoldInteraction)
```

Comment:
The problem I have with this is that I calls out a specific implementation (HoldInteraction) and not a property of the model. Maybe what @bmalrat suggested is similar (not sure), that if something else is added as an extension, e.g. LongHoldInteraction, it breaks again?